projects/step-core/src/lib/shared/function-type-composite.form.ts

```diff
@@ -0,0 +1,20 @@
+import { FormBuilder } from '@angular/forms';
+import { FunctionComposite } from './function-composite.interface';
+
+export type FunctionTypeCompositeForm = ReturnType<typeof functionTypeCompositeFormCreate>;
+
+export const functionTypeCompositeFormCreate = (formBuilder: FormBuilder) => {
+  const formGroup = formBuilder.nonNullable.group({});
+
+  return formGroup;
+};
+
+export const functionTypeCompositeFormSetValueToForm = (
+  form: FunctionTypeCompositeForm,
+  model: FunctionComposite
+): void => {};
+
+export const functionTypeCompositeFormSetValueToModel = (
+  form: FunctionTypeCompositeForm,
+  model: FunctionComposite
+): void => {};
```

Comment:
@bojidaryovchev 
What is the purpose of these functions above?
They actually do nothing